src/dev/build/lib/config.ts

```diff
@@ -104,12 +120,26 @@ export class Config {
   }
 
   /**
-   * Return the list of Platforms we are targeting, if --this-platform flag is
-   * specified only the platform for this OS will be returned
+   * Return true if a supported Platform is specified
+   */
+  hasSpecifiedPlatform() {
+    return this.darwinX64 || this.linuxArm64 || this.linuxX64;
+  }
+
+  /**
+   * Return the list of supported Platforms, if --all-platform flag is specified.
+   * Return the targeted Platform, if --target-platform flag is specified.
+   * If no flag, only the platform for this OS will be returned.
    */
   getTargetPlatforms() {
     if (this.targetAllPlatforms) {
       return ALL_PLATFORMS;
+    } else if (this.darwinX64) {
```

Comment:
Can the user pass it multiple CLI options? If so it will always prioritize darwinx64 over others.

Reply:
Right now user can't pass multiple CLI options. It will prioritize darwin x64 if both darwin and linux are both passed. If we want to accept multiple options, we need to change quite a bit amount of code (nodejs download, create notice and license). The original code actually only download linux-x64 nodejs and create notice and license using only linux-x64. Do we want to support multiple CLI options?

Reply:
Will support multiple CLI options.